packages/nouns-webapp/src/pages/Playground/index.tsx

```diff
@@ -56,7 +56,7 @@ const nounsSDKLink = (
 
 const DEFAULT_TRAIT_TYPE = 'heads';
 
-const encoder = new PNGCollectionEncoder(ImageData.palette);
+const encoder = new PNGCollectionEncoder();
```

Comment:
`ImageData.palette` must be passed into the encoder. Otherwise, playground Nouns will be all black.

Reply:
Whoops. I was def not using my brain when I made this change. My bad. Rectified.

Reply:
No worries. If you made this change because you were getting a type error, rebuilt from the monorepo root to fix it.